[2025-04-04T15:07:25.252+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: download_and_store_images.download_and_upload_images scheduled__2024-02-17T00:00:00+00:00 [queued]>
[2025-04-04T15:07:25.378+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: download_and_store_images.download_and_upload_images scheduled__2024-02-17T00:00:00+00:00 [queued]>
[2025-04-04T15:07:25.383+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2025-04-04T15:07:25.385+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2025-04-04T15:07:25.388+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2025-04-04T15:07:25.503+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): download_and_upload_images> on 2024-02-17 00:00:00+00:00
[2025-04-04T15:07:25.796+0000] {standard_task_runner.py:55} INFO - Started process 82 to run task
[2025-04-04T15:07:26.119+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'download_and_store_images', 'download_and_upload_images', 'scheduled__2024-02-17T00:00:00+00:00', '--job-id', '809', '--raw', '--subdir', 'DAGS_FOLDER/dag_data_collect.py', '--cfg-path', '/tmp/tmpln5ic91d']
[2025-04-04T15:07:26.139+0000] {standard_task_runner.py:83} INFO - Job 809: Subtask download_and_upload_images
[2025-04-04T15:07:26.868+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:249: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-04-04T15:07:27.336+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  return not conf.get("database", "sql_alchemy_conn").startswith("mssql")

[2025-04-04T15:07:28.351+0000] {task_command.py:388} INFO - Running <TaskInstance: download_and_store_images.download_and_upload_images scheduled__2024-02-17T00:00:00+00:00 [running]> on host fd5deae9c635
[2025-04-04T15:07:29.183+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=download_and_store_images
AIRFLOW_CTX_TASK_ID=download_and_upload_images
AIRFLOW_CTX_EXECUTION_DATE=2024-02-17T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-02-17T00:00:00+00:00
[2025-04-04T15:07:33.942+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:07:34.122+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:07:34.335+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/providers/amazon/aws/utils/connection_wrapper.py:8: DeprecationWarning: extra['host'] is deprecated and will be removed in a future release. Please set extra['endpoint_url'] instead
  #

[2025-04-04T15:08:00.681+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:08:00.686+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:08:19.597+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:08:19.677+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:08:24.417+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:08:24.429+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:08:30.846+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:08:30.908+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:08:35.992+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:08:36.004+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:08:40.929+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:08:40.930+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:08:43.552+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:08:43.570+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:08:48.079+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:08:48.081+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:08:56.680+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:08:56.682+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:08:59.325+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:08:59.328+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:09:02.429+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:09:02.441+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:09:06.002+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:09:06.012+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:09:11.032+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:09:11.036+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:09:13.316+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:09:13.331+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:09:18.783+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:09:18.795+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:09:29.458+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:09:29.460+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:09:34.111+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:09:34.113+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:09:37.807+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:09:37.820+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:09:41.507+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:09:41.512+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:09:47.842+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:09:47.847+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:09:53.769+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:09:53.777+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:10:00.086+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:10:00.099+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:10:07.362+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:10:07.377+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:10:10.442+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:10:10.450+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:10:19.227+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:10:19.232+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:10:22.637+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:10:22.671+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:10:25.327+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:10:25.329+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:10:28.151+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:10:28.158+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:10:48.600+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:10:48.739+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:11:14.985+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:11:15.070+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:11:28.147+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:11:28.150+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:11:37.548+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:11:37.573+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:11:52.718+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:11:52.724+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:11:58.353+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:11:58.358+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:12:04.904+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:12:04.909+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:12:10.730+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:12:10.732+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:12:15.812+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:12:15.833+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:12:20.922+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:12:20.927+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:12:30.351+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:12:30.356+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:12:46.822+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:12:46.830+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:12:58.091+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:12:58.093+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:13:04.170+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:13:04.172+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:13:11.322+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:13:11.381+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:13:16.000+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:13:16.003+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:13:20.284+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:13:20.286+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:13:25.027+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:13:25.029+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:13:29.905+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:13:29.914+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:13:36.509+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:13:36.528+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:13:42.893+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:13:42.900+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:13:53.690+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:13:53.696+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:14:07.728+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:14:07.787+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:14:42.278+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:14:42.296+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:16:36.532+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:16:36.664+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:17:22.169+0000] {base_job.py:243} ERROR - LocalTaskJob heartbeat got an exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 3361, in _wrap_pool_connect
    return fn()
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 327, in connect
    return _ConnectionFairy._checkout(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 894, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 493, in checkout
    rec = pool._do_get()
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/impl.py", line 256, in _do_get
    return self._create_connection()
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 273, in _create_connection
    return _ConnectionRecord(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 388, in __init__
    self.__connect()
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 691, in __connect
    pool.logger.debug("Error on connect(): %s", e)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/util/langhelpers.py", line 72, in __exit__
    with_traceback=exc_tb,
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 211, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 686, in __connect
    self.dbapi_connection = connection = pool._invoke_creator(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/create.py", line 578, in connect
    return dialect.connect(*cargs, **cparams)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 598, in connect
    return self.dbapi.connect(*cargs, **cparams)
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: could not translate host name "postgres" to address: Temporary failure in name resolution


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/jobs/base_job.py", line 233, in heartbeat
    session.merge(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 3057, in merge
    _resolve_conflict_map=_resolve_conflict_map,
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 3135, in _merge
    options=options,
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 2856, in get
    execution_options=execution_options,
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 2974, in _get_impl
    load_options=load_options,
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/orm/loading.py", line 534, in load_on_pk_identity
    bind_arguments=bind_arguments,
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 1713, in execute
    conn = self._connection_for_bind(bind)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 1553, in _connection_for_bind
    engine, execution_options
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 747, in _connection_for_bind
    conn = bind.connect()
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 3315, in connect
    return self._connection_cls(self, close_with_result=close_with_result)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 96, in __init__
    else engine.raw_connection()
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 3394, in raw_connection
    return self._wrap_pool_connect(self.pool.connect, _connection)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 3365, in _wrap_pool_connect
    e, dialect, self
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2199, in _handle_dbapi_exception_noconnection
    sqlalchemy_exception, with_traceback=exc_info[2], from_=e
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 211, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 3361, in _wrap_pool_connect
    return fn()
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 327, in connect
    return _ConnectionFairy._checkout(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 894, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 493, in checkout
    rec = pool._do_get()
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/impl.py", line 256, in _do_get
    return self._create_connection()
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 273, in _create_connection
    return _ConnectionRecord(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 388, in __init__
    self.__connect()
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 691, in __connect
    pool.logger.debug("Error on connect(): %s", e)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/util/langhelpers.py", line 72, in __exit__
    with_traceback=exc_tb,
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 211, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/pool/base.py", line 686, in __connect
    self.dbapi_connection = connection = pool._invoke_creator(self)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/create.py", line 578, in connect
    return dialect.connect(*cargs, **cparams)
  File "/home/airflow/.local/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 598, in connect
    return self.dbapi.connect(*cargs, **cparams)
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) could not translate host name "postgres" to address: Temporary failure in name resolution

(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2025-04-04T15:19:12.994+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:19:12.996+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:19:20.720+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:19:20.730+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:19:29.015+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:19:29.029+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:19:34.700+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:19:34.707+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:19:38.525+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:19:38.526+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:19:43.664+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:19:43.679+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:19:47.269+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:19:47.271+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:19:51.644+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:19:51.646+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:19:55.726+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:19:55.729+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:19:58.701+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:19:58.713+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:20:08.331+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:20:08.334+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:20:11.477+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:20:11.499+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:20:16.196+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:20:16.237+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:20:18.558+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:20:18.565+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:20:26.011+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:20:26.016+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:20:33.323+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:20:33.329+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:20:44.617+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:20:44.619+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:20:48.481+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:20:48.487+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:20:55.561+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:20:55.569+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:21:03.671+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:21:03.682+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:21:24.148+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:21:24.324+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:21:45.907+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:21:45.955+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:22:52.618+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:22:52.877+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:23:11.891+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:23:11.900+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:23:37.025+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:23:37.101+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:23:53.342+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:23:53.384+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:24:04.266+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:24:04.269+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:25:47.123+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:25:47.138+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:27:26.895+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:27:26.966+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:28:39.151+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:28:39.306+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:29:13.970+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:29:13.987+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:29:26.212+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:29:26.217+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:29:37.687+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:29:37.693+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:29:47.494+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:29:47.515+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:31:30.077+0000] {base.py:73} INFO - Using connection ID 'minio_s3' for task execution.
[2025-04-04T15:31:32.634+0000] {connection_wrapper.py:334} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-04-04T15:34:18.946+0000] {local_task_job.py:208} INFO - Task exited with return code Negsignal.SIGKILL
[2025-04-04T15:34:27.252+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
